src/dbnode/generated/thrift/rpc.thrift

```diff
@@ -432,6 +433,7 @@ struct AggregateQueryRequest {
 	8: optional TimeType rangeType = TimeType.UNIX_SECONDS
 	9: optional binary source
 	10: optional i64 docsLimit
+	11: optional bool requireExhaustive
```

Comment:
Does this indicate that the results are exhaustive or whether you passed the flag in in the first place? If it's the former, let's just call it `exhaustive`

Reply:
it's on the request side, saying it requires the results to be exhaustive. Would rather stick with that as it's how our other requests use it.

Reply:
Oh wait, my bad, this is a request. Not a result. Misread the struct